modules/mobile/spec/request/facilities_info_request_spec.rb

```diff
@@ -56,7 +66,7 @@
         it 'returns facility details sorted by closest to user\'s home' do
           VCR.use_cassette('mobile/appointments/get_multiple_mfs_facilities_200',
                            match_requests_on: %i[method uri]) do
-            get('/mobile/v0/facilities-info/home', headers: sis_headers, params:)
```

Comment:
Removing all these params because the params (defined at the top of the file) are specific to sorting by the user's current location.